lusidtools/cocoon/dateorcutlabel.py

```diff
@@ -8,14 +8,33 @@
 
 
 class DateOrCutLabel(UserString):
-    def __init__(self, datetime_value):
-        def convert_datetime_utc(datetime_value):
+    def __init__(self, datetime_value, custom_date_format=None):
+        def convert_datetime_utc(datetime_value, custom_date_format=None):
             """
             This function ensures that a variable is a timezone aware UTC datetime
 
             :param any datetime_value:
             :return: datetime datetime_value: The converted timezone aware datetime in the UTC timezone
             """
+            if custom_date_format:
+                if not isinstance(datetime_value, str):
+                    raise TypeError(f"Date {datetime_value} is of type {type(datetime_value)} must be of type 'str' "
+                                    f"when specifying a custom date format. ")
+
+                try:
+                    datetime_value = pd.to_datetime(datetime_value, format=custom_date_format,
```

Comment:
shouldn't this be `utc=True`?

Reply:
UTC is irrelevant at this stage as it is set later on line 37. I have removed the argument altogether to reduce conduction.
 
Datetime value is passed into `formatdt()` which extracts "%Y-%m-%d %H:%M:%S.%f" and then formats it to UTC
```
np.datetime_as_string(
                                        arr=np.datetime64(dt.strftime("%Y-%m-%d %H:%M:%S.%f")), timezone="UTC", unit="us"
                                    )
```